src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoComplete/tests/AutoComplete.test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
```

Comment:
Would be nice to actually use flow in here, so that we see if there are no errors 😃